cmd/minishift/cmd/start.go

```diff
@@ -892,3 +895,13 @@ func startTray() error {
 	}
 	return nil
 }
+
+func setStaticIP(hostVm *host.Host) {
+	fmt.Print("-- Writing current configuration for static assignment of IP address ... ")
+	_, err := minishiftNetwork.ConfigureStaticAssignment(hostVm.Driver)
+	if err != nil {
+		fmt.Println("WARN")
```

Comment:
We should print the `err` inside of `if glog.V(2)` here. It will help users to understand why they are seeing WARN.